health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -0,0 +1,270 @@
+import { useHistory } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Button, Card, InfoButton, Loader, TableMolecule, TextInput, Toast, TooltipWrapper } from "@egovernments/digit-ui-components";
+import { CustomSVG } from "@egovernments/digit-ui-components";
+import { CheckBox } from "@egovernments/digit-ui-components";
+import axios from "axios";
+
+export const downloadExcelWithName = ({ fileStoreId = null, customName = null }) => {
+    const downloadExcel = (blob, fileName) => {
+        const link = document.createElement("a");
+        link.href = URL.createObjectURL(blob);
+        link.download = fileName + ".xlsx";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)